src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/dao/HttpGoogleBillingDAO.scala

```diff
@@ -134,26 +141,16 @@ class HttpGoogleBillingDAO(appName: String, serviceAccountPemFile: String, billi
     // there is not much else to be done... too bad scala does not have duck typing.
     operation.operationType match {
```

Comment:
The two cases seem to have a lot of repeated code

Reply:
that's what the comment above says